.github/actions/upload-bundles/action.yml

```diff
@@ -41,6 +41,10 @@ runs:
       id: bundles
       run: |
         # Rename bundles to consistent names
+        echo "build/${{ inputs.platform }} contents..."
```

Comment:
This (and more below) looks like debug code that could/should be removed once you have made sure the builds are working.